dockstore-webservice/src/main/java/io/dockstore/webservice/resources/TokenResource.java

```diff
@@ -301,135 +297,150 @@ public Token addToken(@ApiParam("code") String satellizerJson) {
     /**
      * Adds a Google token to the existing user if user is authenticated already.
      * Otherwise, create a new Dockstore account too and also add token
-     * @param authUser      The optional Dockstore-authenticated user
-     * @param satellizerJson    Satellizer object returned by satellizer
-     * @return              The user's Dockstore token
+     *
+     * @param authUser       The optional Dockstore-authenticated user
+     * @param satellizerJson Satellizer object returned by satellizer
+     * @return The user's Dockstore token
      */
     @POST
     @Timed
     @UnitOfWork
     @Path("/google")
-    @ApiOperation(value = "Allow satellizer to post a new Google token to dockstore", authorizations = { @Authorization(value = JWT_SECURITY_DEFINITION_NAME) },
-            notes = "A post method is required by satellizer to send the Google token",
-            response = Token.class)
+    @ApiOperation(value = "Allow satellizer to post a new Google token to dockstore", authorizations = {
+            @Authorization(value = JWT_SECURITY_DEFINITION_NAME) }, notes = "A post method is required by satellizer to send the Google token", response = Token.class)
     public Token addGoogleToken(@ApiParam(hidden = true) @Auth Optional<User> authUser, @ApiParam("code") String satellizerJson) {
         Gson gson = new Gson();
         JsonElement element = gson.fromJson(satellizerJson, JsonElement.class);
         JsonObject satellizerObject = element.getAsJsonObject();
-
         final String code = satellizerObject.get("code").getAsString();
         final String redirectUri = satellizerObject.get("redirectUri").getAsString();
         String accessToken = null;
         String refreshToken = null;
         if (accessToken == null) {
```

Comment:
accessToken is set to null two lines up, no need to check for null again (not changed, but noticed in trying to understand code).